altos-rust/port/cortex-m0/src/peripheral/usart/control.rs

```diff
@@ -201,36 +442,69 @@ impl CR1 {
 /// CR2
 // ------------------------------------
 
-/// There are four stop bit settings: .5, 1, 1.5, 2
-#[derive(Copy, Clone)]
+/// Defines the possible StopLength configurations for the Usart.
+/* Half: 0.5 stop bit
+ * One: 1 stop bit
+ * OneAndHalf: 1.5 stop bits
+ * Two: 2 stop bits
+ */
+#[derive(Copy, Clone, Debug)]
 pub enum StopLength {
     Half,
     One,
     OneAndHalf,
     Two,
 }
 
-#[derive(Copy, Clone)]
+/// Defines the base address variable for control register two.
```

Comment:
Again, doc comments are not strictly necessary but might want more detail on what it's purpose is